UPGRADING.md

```diff
@@ -115,6 +115,8 @@
     - `Psalm\Type\Union`
  - While not a BC break per se, all classes / interfaces / traits / enums under
    `Psalm\Internal` namespace are now marked `@internal`.
+ - To remove a variable from the context, Context::remove(). Calling
+   `unset($context->vars_in_scope[$var_id])` can cause problems when using references.
```

Comment:
Kind of makes me want `@psalm-external-readonly` that would be like `@psalm-readonly-allow-private-mutation` but instead of restricting modifications to the class itself, it would restrict it to a namespace.